solidity/random-beacon/contracts/libraries/DKG.sol

```diff
@@ -205,6 +205,24 @@ library DKG {
         );
         require(!hasDkgTimedOut(self), "dkg timeout already passed");
 
+        (uint256 firstEligibleIndex, uint256 lastEligibleIndex) = Submission
+            .getEligibilityRange(
+                self.seed,
```

Comment:
According to the RFC, the "seed" for eligibility range calculations should be the hash of new group public key (see https://github.com/keep-network/keep-core/blob/main/docs/rfc/rfc-19-random-beacon-v2.adoc#22-fees-and-rewards) not the seed of the DKG itself. Parameter name in `getEligibilityRange` is a little bit confusing.

Reply:
Try/catch works only for failed external calls and contract creation calls (source: https://docs.soliditylang.org/en/latest/control-structures.html#try-catch)

Reply:
Nice catch! 6bf451d14b890020408dd3a06873a5950a845f47